synapse/tests/test_lib_types.py

```diff
@@ -353,6 +361,15 @@ def test_syntag(self):
         self.eq('foo', subs.get('up'))
 
         self.raises(s_exc.BadTypeValu, tagtype.norm, '@#R)(Y')
+        self.raises(s_exc.BadTypeValu, tagtype.norm, 'foo\udcfe.bar')
+        self.raises(s_exc.BadTypeValu, tagtype.norm, 'foo.')
+        self.raises(s_exc.BadTypeValu, tagtype.norm, '.')
+        self.raises(s_exc.BadTypeValu, tagtype.norm, '')
+        # Okay behavior?
+        # self.eq('icon.ॐ', tagtype.norm('ICON.ॐ')[0])
```

Comment:
I'm leaving these here since there is a bit of an open-ended question of how we want tag normalization to behave for non-ascii charsets

Reply:
I think that these tests should pass and that we should uncomment them